packages/daisy/src/components/pagination/pagination.tsx

```diff
@@ -0,0 +1 @@
+export {};
```

Comment:
This file should wrap the headless one.
here is an example https://github.com/qwikifiers/qwik-ui/blob/main/CODING_STANDARDS.md#example-of-headless-button-customisation-with-daisy